pkg/controller/utils/clusterinfo.go

```diff
@@ -127,6 +127,50 @@ func GetClusterMasterInstances(kclient client.Client) ([]string, error) {
 	return ids, nil
 }
 
+// MasterInstance used to fill out TargetDescription
+// when calling registerTargetInput
+type MasterInstance struct {
+	AvailabilityZone string
+	IPaddress        string
+}
+
+// GetMasterInstancesAZsandIPs gets all the master instances AZs and IPs
+func GetMasterInstancesAZsandIPs(kclient client.Client) ([]MasterInstance, error) {
+	machineList := &machineapi.MachineList{}
+	listOptions := []client.ListOption{
+		client.InNamespace("openshift-machine-api"),
+		client.MatchingLabels{masterMachineLabel: "master"},
+	}
+	err := kclient.List(context.TODO(), machineList, listOptions...)
+	if err != nil {
+		return []MasterInstance{}, err
+	}
+
+	masterInstances := make([]MasterInstance, 0)
```

Comment:
you should initialize this as well like it was mentioned in the function of the awsclient